server/graphql_schemas/views.py

```diff
@@ -3,16 +3,14 @@
 from graphene_django.views import GraphQLView
```

Comment:
Why is this still here?

Reply:
My python is shaky but I assumed since the lower function was still there and using GraphQLView, I would still keep it. Open to suggestion on how I can get rid of it or if I need to.